keys.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def check_keys():
+    keys_path = './better_weather_keys.txt'
```

Comment:
Please use [Python constants](https://realpython.com/python-constants/#what-constants-are) for such non-changeable variables.

Reply:
Done.